libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/Edge.kt

```diff
@@ -0,0 +1,155 @@
+package com.mapbox.navigation.base.trip.model
+
+import com.mapbox.geojson.LineString
+
+/**
+ * Basic Edge class
+ */
+data class Edge(
+    /**
+     * The Edge identifier.
+     */
+    var id: Int,
+
+    /**
+     * The level of the Edge (0 being the mpp, 1 branches of the mpp,
+     * 2 branches of level 1 branches, etc)
+     */
+    var level: Int,
+
+    /**
+     * The probability of the Edge in percentage.
+     */
+    var probability: Double,
+
+    /**
+     * The bearing in degrees clockwise at the start of the edge.
+     */
+    var heading: Double,
+
+    /**
+     * The Edge's length in meters.
+     */
+    var length: Double,
+
+    /**
+     * The outgoing Edges.
+     */
+    var out: List<Edge> = emptyList(),
+
+    /**
+     * The edge's functional road class
+     */
+    var frc: String,
+
+    /**
+     * The edge's way IDs
+     */
+    var wayId: List<String>,
+
+    /**
+     * Specifies if the geometry is in the forward or
+     * backwards direction
+     */
+    // TODO Not returned by NN
+    // var positiveDirection: Boolean = true,
```

Comment:
Because as I understand it refers to this field in routing tiles https://github.com/mapbox/valhalla-internal/blob/e697397d0d370c541da5fa63bd92dbdeee90c923/valhalla/baldr/directededge.h#L360
It is needed to know in which direction we must "read" shape of edge in routing tiles(to save memory Valhalla stores shapes for both sides of street once, but edge only contains reference to this shape and `forward` flag which indicates from which end this shape should be used). Since here we always return shape of edge with edge itself we can just always return shape in "forward" direction. 
But it was just my understanding of this field, probably it could mean something different.

Reply:
This will be exposed *positiveDirection: Boolean*